src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -170,6 +170,13 @@ export class CalciteRadioButton {
    */
   @Event() calciteRadioButtonChange: EventEmitter;
 
+  /** Fires when the checked property changes.  This is an internal event used for styling purposes only.
+   * Use calciteRadioButtonChange or calciteRadioButtonGroupChange for responding to changes in the checked value for forms.
+   * @internal
+   */
+  @Event() calciteRadioButtonCheckedChange: EventEmitter;
```

Comment:
I wonder if you could enforce this event being internal by setting `composed` or `bubbles` to false in the EventOptions?

Reply:
It needs to bubble up for `tile-select` to be able to use it.  What does `composed` mean?

Reply:
oh in that case we can just `stopPropagation` in `tile-select` (you may be already doing this).

I think `composed` is how you control whether the event can get through shadow dom or not.

Reply:
I'll try that.  I also looked up `composed` and tried that as well, but I was still able to listen for the event at the document level, so it doesn't appear to work anyway.